pom.xml

```diff
@@ -149,6 +149,24 @@ limitations under the License.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
```

Comment:
What is this for?

Reply:
This is for automatically adding avro-generated codes to IntelliJ without any configuration. I will make a separate PR for this.